packages/babel-plugin-transform-runtime/src/definitions.js

```diff
@@ -1,104 +1,141 @@
-export default {
-  builtins: {
-    Symbol: "symbol",
-    Promise: "promise",
-    Map: "map",
-    WeakMap: "weak-map",
-    Set: "set",
-    WeakSet: "weak-set",
-    setImmediate: "set-immediate",
-    clearImmediate: "clear-immediate",
-    parseFloat: "parse-float",
-    parseInt: "parse-int",
-  },
+import semver from "semver";
 
-  methods: {
-    Array: {
-      from: "array/from",
-      isArray: "array/is-array",
-      of: "array/of",
-    },
+const getDefinitions = runtimeVersion => {
```

Comment:
You can do `export default runtimeVersion => {` to avoid splitting up the export and the function.